### ISSUE TITLE
children method does not respond to empty?

### ISSUE THREAD

**Issue by catmando** [ruby-hyperloop/hyper-react#181]
children.empty? 

Seems to be just the way ruby works?  perhaps there is a better way to define an enumerable?

**Comment by zetachang** [comment]
I think there is no reason to stop us from adding `empty?` to `React::Children`

**Comment by catmando** [comment]
yeah I'm okay with that... but it seems odd.  Perhaps there is a different way to create an enumerable we should be using?  The reason I say that, is what else are we missing that we should be getting?

**Comment by zetachang** [comment]
Ruby's `Enumerable` doesn't provide `empty?` method, it got something similar which is `any?`.
